### PR TITLE
Adjusting for postmates usage

### PR DESCRIPTION
**Pull request by doubleyou**
- Use us-west-1 by default for now.
- Make URLs from inside `index.html` work. Our S3 bucket is available from inside the VPC, so there's no need in ACL or even HTTPS.

### CODE REVIEW

**Comment by doubleyou** [comment]
@postmates/infra r?

File: mkwheelhouse.py [hunk 1]
```diff
@@ -68,7 +68,9 @@ def has_key(self, key):
 
     def generate_url(self, key):
         key = self.get_key(key)
-        return key.generate_url(expires_in=0, query_auth=False)
+        url = key.generate_url(expires_in=0, query_auth=False)
+        # Temporary hack for fixing the links
+        return url.replace('https', 'http').split('?')[0]
```
Comment (aabhassharma) [inline mkwheelhouse.py:73]: heh

**Comment by aabhassharma** [comment]
r+

File: mkwheelhouse.py (hunk 1 above)
Comment (doubleyou) [inline mkwheelhouse.py:73]: I mean, I don't even know why boto throws in ACL parameters there :) But they just don't work.

**Pull request merged**